.github/workflows/workflow.yaml

```diff
@@ -90,13 +123,13 @@ jobs:
             bazel build //docs:api_html
             bazel build //docs:guide_html
             '
-      - name: Build & test - rules_haskell_nix
+      - name: Build - rules_haskell_nix
         if: matrix.module == 'rules_haskell_nix'
         run: |
           nix-shell --arg docTools false --argstr ghcVersion ${{ matrix.ghc }} --pure --run '
             set -euo pipefail
             cd rules_haskell_nix
-            bazel test //...
+            bazel build //...
```

Comment:
I guess this is done because `rules_haskell_nix` does not contain any tests (apart from the manual buildifier test)?
We need not to forget to switch this back if we every add tests again.
I wonder if it may be safer to add a dummy test and keep this as `bazel test //...`.